test/db_test.py

```diff
@@ -61,6 +61,21 @@ def noUserFound():
 			user = User.get_by_name('badname')
 			assert_that(user, is_(none()))
 
+		@it('User accessed_at field updates (And nothing else)')
+		def userAccessedAt():
+			before = User.get_by_name(test_name).__dict__.get('__data__')
```

Comment:
You don't need to access the special fields like this. You're doing this so you can do a dictionary comparison later, right? You can instead to this:
```
from playhouse.shortcuts import model_to_dict
...
before = model_to_dict(User.get_by_name(test_name))
```